### DISCUSSION TITLE
PostgreSQL version

### DISCUSSION THREAD

**Discussion by motionthings** [goauthentik/authentik#3248] [Q&A] [answered]
Is there a preferred PostgreSQL version or will any version work?

**Comment by BeryJu** [comment, marked as answer]
I develop, test and run with Postgres 12 so that and anything after it should work, previous versions should work too in theory as long as they support JSON fields and recursive queries

**Reply by sevmonster** [reply]
I have always run authentik with Postgres and have only had issues with the database when doing dirty upgrades to other dev versions (my own fault) :)

I think I started on 11.
